src/components/playlist-modal/Modal.css

```diff
@@ -0,0 +1,31 @@
+.playlist-modal {
+	display: block;
+	z-index: 10;
+	background-color: #fff;
+	width: 30rem;
+	max-height: 32rem;
+	position: absolute;
+	top: 40%;
+	display: flex;
+	flex-direction: column;
+	border-radius: 10px;
```

Comment:
Seems like you are using position:absolute and flex. This might create a problem in future. Just be carefull

Reply:
fixed: removed display:block.